collector/textfile.go

```diff
@@ -249,3 +242,19 @@ fileLoop:
 	)
 	return nil
 }
+
+// helper: returns true if file contains unsupported timestamps
```

Comment:
style nit: We prefer sentence-style comments starting with upper-case letters and ending with a period.

I would omit the "helper:" prefix too.

Reply:
ok nice